app/controllers/spotlight/catalog_controller.rb

```diff
@@ -8,6 +8,8 @@ module Spotlight
   # rubocop:disable Metrics/ClassLength
   class CatalogController < ::CatalogController
     include Spotlight::Concerns::ApplicationController
+    helper_method :breadcrumbs
+
```

Comment:
I don't think you need this anymore now that `helper_method :breadcrumbs` is set in `Spotlight::Controller`
```suggestion
```